examples/notebooks/cache_dataset_speed.ipynb

```diff
@@ -180,11 +180,10 @@
     "            loss.backward()\n",
     "            optimizer.step()\n",
     "            epoch_loss += loss.item()\n",
-    "            print('{}/{}, train_loss: {:.4f}, step time: {:.4f}'.format(\n",
-    "                  step, len(train_ds) // train_loader.batch_size, loss.item(), time.time() - step_start))\n",
+    "            print(f\"{step}/{len(train_ds) // train_loader.batch_size}, train_loss: {loss.item()}, step time: {time.time() - step_start}\")\n",
     "        epoch_loss /= step\n",
     "        epoch_loss_values.append(epoch_loss)\n",
-    "        print('epoch {} average loss: {:.4f}'.format(epoch + 1, epoch_loss))\n",
+    "        print(f\"epoch {epoch + 1} average loss: {epoch_loss}\")\n",
```

Comment:
The previous string only keeps 4 places after the decimal point.
Something like: `f"XXX{data: 4}"`

Thanks.